app/assets/stylesheets/application.css

```diff
@@ -73,14 +73,32 @@ ul .inline {
   margin-top: 40px;
 }
 
-select#submission_event_id {
+select#submission_event_instance_id {
   display: block;
 }
 
 .g-recaptcha {
   margin-bottom: 1.5rem;
 }
 
+fieldset.fieldset {
+  border: 1px solid #c0c0c0;
+  border-radius: 3px;
+  margin: 0 2px;
+  margin-bottom: 15px;
+  padding: 0.5em 0.8em 0.75em;
+}
+
+/**
+ * 1. Correct `color` not being inherited in IE 8/9/10/11.
```

Comment:
@flyinggrizzly — are you saying that the change below allows compatibility with IE 8,9, etc..or that it's not working as it is..?

Reply:
This fieldset code is pretty much [straight out of Normalize](https://github.com/necolas/normalize.css/blob/4ab3de5bdd26b161c3c82a5a2f72df3e57a8e4bf/normalize.css#L227), which I added because I wanted to override what I'm guessing where Skeleton styles where the fieldset legend wasn't inline with the fieldset border or was getting squashed in there (it looked pretty rough).

The comment comes with it, though that link is to what looks to be a newer version of Normalize. My read is that it fixes some weird behaviors with fieldset legends in IE. Skeleton may resolve those in other ways, but for this look I think it's necessary.

That said, CSS is *not* my strong suit...

Reply:
Okay, cool! 

We'll leave it as is for now.

Thanks for the explanation.